### ISSUE TITLE
Error during backup: "nodetool clearsnapshot -t medusa-.... returned non-zero exit status 1

### ISSUE THREAD

**Issue by nicholasamorim** [thelastpickle/cassandra-medusa#42]
I'm trying to backup my cluster for the 3rd time
and all of them have ended with something like ERROR: This error happened during the backup: 

```
Command '['nodetool', 'clearsnapshot', '-t', 'medusa-0d88c264-3149-42ba-b71e-d3d5a14ca1d0']' returned non-zero exit status 1.
```

This seems to happen in all nodes (6 in total). I'm not even sure where to start debugging this.

**Comment by nicholasamorim** [comment]
The error is pretty consistent, it happened in all 6 nodes. Funny that it does show it being uploaded, then errors right after.

```
[2019-12-18 10:33:07,196] INFO: Uploading /cassandradb/data/sm_test/download_resource-2b7b7b609fd611e79c8dd1d1eb8370ed/snapshots/medusa-b6cd16fe-62ae-47fb-9997-34e7a83ee8d2/md-62-big-CompressionInfo.db (1.167KiB)
[2019-12-18 10:33:07,224] INFO: Uploading /cassandradb/data/sm_test/keyword_detection_by_time_per_alert-373e5a008f5711e89a83ebad2915d063/snapshots/medusa-b6cd16fe-62ae-47fb-9997-34e7a83ee8d2/schema.cql (1.104KiB)
[2019-12-18 10:33:07,327] ERROR: This error happened during the backup: Command '['nodetool', 'clearsnapshot', '-t', 'medusa-b6cd16fe-62ae-47fb-9997-34e7a83ee8d2']' returned non-zero exit status 1.
```

**Comment by nicholasamorim** [comment]
In my case the error was happening due to lack of memory to run `nodetool` as a mega repair was happening at the same time.

Still, there was some room for improvement as the backup was essentially finished.

Now the backup does not fail if the `clearsnapshot` fail, but rather logs a warning for you to clean the snapshots manually. If you set the log level to DEBUG you will also be able to see the exact error message that nodetool returned.

This was merged in #44
